howfairis/__init__.py

```diff
@@ -17,6 +18,6 @@
     "cli",
     "Compliance",
     "Platform",
-    "Repo"
-
+    "Repo",
+    "workarounds"
```

Comment:
Dont think workaround needs to be public. All workaounds are called directly/indirectly from a the public API.
The workarounds are internal to the library.
```suggestion
    "Repo"
```

Reply:
Agree will change